src/extension/background-script/actions/nostr/helpers.ts

```diff
@@ -0,0 +1,44 @@
+import * as secp256k1 from "@noble/secp256k1";
+import createHash from "create-hash";
```

Comment:
can we use crypto-js for the hash?  
so far `create-hash` is an indirect dependency I think. crypto-js is a direct dependency.